src/ToricVarieties/LineBundles/constructors.jl

```diff
@@ -0,0 +1,59 @@
+########################
+# 1: The Julia type for toric line bundles
+########################
+
+abstract type ToricCoherentSheaf end
+
+struct ToricLineBundle <: ToricCoherentSheaf
+            variety::AbstractNormalToricVariety
+            divisor_class::GrpAbFinGenElem
+end
+export ToricLineBundle
+
+
+########################
+# 2: Generic constructors
+########################
+
+
+@doc Markdown.doc"""
+    ToricLineBundle(v::AbstractNormalToricVariety, c::Vector{fmpz})
+
+Construct the line bundle on the abstract normal toric variety `v` with class `c`.
+```
+"""
+function ToricLineBundle(v::AbstractNormalToricVariety, input_class::Vector{fmpz})
+    matrix = AbstractAlgebra.matrix(ZZ, hcat( Matrix{fmpz}(undef, length(input_class), 0), input_class))
+    class = GrpAbFinGenElem(picard_group(v), matrix)
```

Comment:
```suggestion
    class = picard_group(v)(input_class)
```

Reply:
This is the preferred way to construct an element of an abelian group given the "coordinates".

Reply:
Thank you. Since this is a small PR, I have just rebased with this change.